python/dask_cudf/.coveragerc

```diff
@@ -1,4 +1,3 @@
 # Configuration file for Python coverage tests
 [run]
 include = dask_cudf/*
```

Comment:
```suggestion
source = dask_cudf
```

Reply:
To align with previous conversation: https://github.com/rapidsai/cudf/pull/10347#discussion_r813554501